src/server/service/match.ts

```diff
@@ -325,6 +340,23 @@ export class Match {
             score: sql`${users.score} + ${player.score}`,
           })
           .where(eq(users.id, player.userId));
+
+        player.achievements
+          .filter((achievement) => {
+            return (
+              achievement === "beginnersLuck" ||
+              achievement === "realHuman" ||
+              achievement === "firstTimer" ||
+              achievement === "busterStreak"
+            );
```

Comment:
Make a constant array of these and check if it includes the achievement instead of doing ors over ors